new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Installer.java

```diff
@@ -35,25 +37,25 @@ public boolean download() {
     boolean downloadResult = true;
     boolean unzipResult = true;
     if (params.verify()
-        && new File(downloadDir, params.fileName()).exists()) {
+        && new File(DOWNLOAD_DIR, params.fileName()).exists()) {
       logger.info("File " + params.fileName() + " already exists.");
     } else {
       checkDirectory(WORK_DIR);
       downloadResult = executeAndVerify(buildDownloadCommand(), params.direct());
     }
-    if (!(new File(downloadDir + "../imagetool/imagetool.sh").exists()) 
+    if (!(new File(IMAGE_TOOL).exists()) 
         && params.unzip()) {
-      unzipResult = unzip(downloadDir, params.fileName(), downloadDir + "/..");
+      unzipResult = unzip();
     }
     return downloadResult && unzipResult;
   }
 
-  private boolean unzip(String path, String fileName, String targetDir) {
+  private boolean unzip() {
     String command = 
```

Comment:
I think `String.format()` would make this easier to read

Reply:
yes. Changed the code to use String.format().